nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/MainHelper.java

```diff
@@ -107,6 +118,16 @@ private static int getMinorJdkVersion() {
         }
     }
 
+    private static int getSecurityJdkVersion() {
+        String jv = System.getProperty("java.version");
```

Comment:
Avoid using abbreviations like "jv", just use "javaVersion" or "jdkVersion" etc here.

Reply:
There is also already JDK.java class in nucleus/common to take care of things like this